opentelemetry-sdk/src/opentelemetry/sdk/environment_variables/__init__.py

```diff
@@ -212,6 +292,9 @@
 OTEL_EXPORTER_OTLP_TRACES_TIMEOUT = "OTEL_EXPORTER_OTLP_TRACES_TIMEOUT"
 """
 .. envvar:: OTEL_EXPORTER_OTLP_TRACES_TIMEOUT
+
+The :envvar:`OTEL_EXPORTER_OTLP_TRACES_TIMEOUT` is the maximum time the OTLP exporter will wait for each batch export.
+
```

Comment:
Missing `OTEL_EXPORTER_JAEGER_CERTIFICATE` and `OTEL_EXPORTER_JAEGER_AGENT_SPLIT_OVERSIZED_BATCHES`?